code/src/containers/tables/orders-table/components/orders-table-body/OrdersTableBody.scss

```diff
@@ -12,4 +12,23 @@
   &-status {
     width: 75px;
   }
+
+  &-status-select-input {
+    padding-right: 14px !important;
+  }
+
+  &-status-select {
+    width: fit-content !important;
+    min-width: auto !important;
+  }
+}
+
+.status-select {
+  padding: 0 !important;
+
+  * {
+    border: none !important;
+    outline: none !important;
+    box-shadow: none !important;
+  }
```

Comment:
Why there is `!important` everythere?

Reply:
Thanks, I used it to overwrite mui. I added comments and removed some of redundant `!important`